File: retaguarda-software-house/delphi/retaguarda_sh/Model/Sincroniza/ObjetoSincroniza.pas
unit ObjetoSincroniza;

interface

uses
  MVCFramework.Serializer.Commons, ModelBase;

type

  [MVCNameCase(ncLowerCase)]
  TObjetoSincroniza = class(TModelBase)
  private
    FTabela: string;
    FRegistros: string;

  public
		[MVCNameAsAttribute('tabela')]
		property Tabela: string read FTabela write FTabela;
		[MVCNameAsAttribute('registros')]
		property Registros: string read FRegistros write FRegistros;

  end;

implementation

{ TEmpresa }



end.

File: retaguarda-software-house/node/retaguarda-sh/src/sincroniza/objeto.sincroniza.ts
export class ObjetoSincroniza {

	tabela: string;
	registros: string;
		
	/**
	* Constructor
	*/
	constructor(objetoJson: {}) {
		if (objetoJson != null) {
			this.tabela = objetoJson['tabela'];
			this.registros = objetoJson['registros'];
		}
	}
    
}